src/Breadcrumb/ThunderNodeEditBreadcrumbBuilder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\thunder\Breadcrumb;
+
+use Drupal\Core\Breadcrumb\Breadcrumb;
+use Drupal\Core\Breadcrumb\BreadcrumbBuilderInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Class to define the breadcrumb builder.
+ */
+class ThunderNodeEditBreadcrumbBuilder implements BreadcrumbBuilderInterface {
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(RouteMatchInterface $route_match) {
+    return in_array($route_match->getRouteName(), [
```

Comment:
This does not work for me on node edit, this applies method is not even called on the edit form

Reply:
The reason is, that  ThunderArticleBreadcrumbBuilder.php wins in that case.
We should consolidate both builders

Reply:
I disagree, ThunderArticleBreadcrumbBuilder should be more specific